src/test/java/SeleniumWebSiteTest.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+
+public class SeleniumWebSiteTest {
+    @Test
+    public void testSeleniumWebDriver() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver");
+        WebDriver driver = new ChromeDriver();
+        try {
+
+            driver.manage().window().maximize();
+            driver.get("https://www.selenium.dev/");
+            driver.findElement(By.xpath("//a[contains(text(),'Downloads')]")).click();
```

Comment:
Add here some waiting action, so page will have a time for loading.

Reply:
In future, will we have another way to assure that page is loaded fully, without `wait()`, not to corrupt test execution time?

Reply:
Yes, it's in future, but for now you should write Thread.sleep

Reply:
Addressed in 03267df.